spec/models/applicant_spec.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Applicant do
```

Comment:
Should test happy path here as well (successful creation of a valid applicant object)